go/staking/api/api.go

```diff
@@ -352,6 +352,10 @@ const (
 	KindStorage   ThresholdKind = 3
 
 	KindMax = KindStorage
+
+	// LogEventGeneralAdjustment is a log event value that signals adjustment
+	// of an account's general balance due to a roothash message.
+	LogEventGeneralAdjustment = "staking/general_adjustment"
```

Comment:
I would move this to the const block at the top as this one is really threshold-kind-related.

Reply:
ok